src/ILCompiler.Compiler/src/IL/HelperExtensions.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
SystemModule/ModuleDesc got trickier than I expected. I stashed the changes to a separate branch - we can discuss it tomorrow.